distros/spring/src/main/java/com/connexta/transformation/service/TransformController.java

```diff
@@ -59,13 +80,64 @@ public TransformController(
    * handles: - Validating the message - Forwarding the request to the service lookup service.
    * Returns an HTTP Status Code of 202 Accepted on success; otherwise, an error status code.
    */
+  @ApiOperation(
+      value = "Submit transformation request",
+      nickname = "transform",
+      notes =
+          "A request to transform resources into discovery metadata and other supporting products.",
+      tags = {"transform"})
+  @ApiResponses({
+    @ApiResponse(
+        code = 201,
+        message =
+            "The transformation request was accepted for processing. The URI for polling the status is returned in the Location header of the response. "),
+    @ApiResponse(
+        code = 400,
+        message =
+            "The client message could not be understood by the server due to invalid format or syntax.",
+        response = ErrorResponse.class),
+    @ApiResponse(
+        code = 401,
+        message = "The client could not be authenticated.",
+        response = ErrorResponse.class),
+    @ApiResponse(
+        code = 403,
+        message = "The client is not authorized.",
+        response = ErrorResponse.class),
+    @ApiResponse(
+        code = 500,
+        message =
+            "The server encountered an unexpected condition that prevented it from fulfilling the request.",
+        response = ErrorResponse.class),
+    @ApiResponse(
+        code = 501,
+        message =
+            "The requested API version is not supported and therefore not implemented. Possible codes reported are: - 501001 - Unable to parse *Accept-Version* - 501002 - The provided major version is no longer supported - 501003 - The provided major version is not yet supported by the server - 501004 - The provided minor version is not yet supported by the server ",
+        response = ErrorResponse.class),
+    @ApiResponse(
+        code = 200,
+        message = "Any other possible errors not currently known.",
+        response = ErrorResponse.class)
+  })
   @RequestMapping(
-      value = "/transform",
-      consumes = {MediaType.APPLICATION_JSON_VALUE},
-      method = RequestMethod.POST)
+      value = {"/transform"},
+      produces = {"application/json"},
```

Comment:
❗️ Use the constant `MediaType.APPLICATION_JSON_VALUE` as opposed to hardcoding. Make sure we do as little change as possible to the interface generated by OpenApi. This will make maintenance much easier when we change the spec.

Reply:
This was a copy-paste from the TrasformApi generated class just to get the class to conform to not implementing that class anymore. Someone else will be taking care of this method. But I can change for now if wanted.

Reply:
As stated in my previous comment, let's keep this in an interface managed inside the API repo. It could be that we split the normally generated interface into 4 new ones. Having it there will allow us to review it alongside the actual generated interface and only revise it when the API change.

Reply:
If we still want to throw the `TransformationException` in order to be handled later for all the endpoints, we may need to update the API to include those as well.

Reply:
Good question. I don't think we can have the API depend on our own repo as that would cause a circular dependency. I would suggest we declare in the interface that we can throw `Exception` and in our actual implementation, only declare the checked exceptions we are actually throwing. A class implementing an interface or extending a base class can always reduce the exceptions it throws but never increase them.

Reply:
This was handled in this PR: https://github.com/connexta/ion-transformation-api/pull/59